### PR TITLE
Remove unused imports/functions

### PR DESCRIPTION
**Pull request by matthiasbeyer**
This PR removes unused stuff.

Lets see what CI tells us about me deleting code :smile:

### CODE REVIEW

File: src/common/setup.rs [hunk 1]
```diff
@@ -21,37 +21,6 @@ macro_rules! asset_map {
     }
 }
 
-pub mod install {
```
Comment (a-kenji) [inline src/common/setup.rs:24]: This is actually [still used](https://github.com/zellij-org/zellij/blob/a24c7f79f1ac40f1e8554aa1615b0a7e020c7ca5/src/server/mod.rs#L190), but good catch! 
I guess the CI should check with --all_features and maybe also just the default features?

Or we need to address this issue in the makefile.

**Review by a-kenji — CHANGES_REQUESTED** [review]
Thanks for this pr! I think this shows that we need to adjust our tests better for conditional compilation.

File: src/common/setup.rs (hunk 1 above)
Comment (matthiasbeyer) [reply src/common/setup.rs:24]: Yep, I guess there's room for improvement with the testsuite then :laughing:

Want me to remove this patch from the patchset?

Comment (a-kenji) [reply src/common/setup.rs:24]: Yes, please do so.

**Comment by matthiasbeyer** [comment]
After rebasing to latest master, this is what remains! :smile:

**Review by a-kenji — APPROVED** [review]
Thank you!

**Pull request merged**